File: nutikas2-main/App.BLL/Services/ServiceService.cs
using App.Contracts.BLL.Services;
using App.Contracts.DAL.Repositories;
using App.Domain.Enums;
using AutoMapper;
using Base.BLL;
using Base.Contracts.DAL;

namespace App.BLL.Services;

public class ServiceService: BaseEntityService<App.Domain.Service, App.BLL.DTO.Service, IServiceRepository>,
IServiceService
{
    public ServiceService(IUnitOfWork uoW, IServiceRepository repository, IMapper mapper) 
        : base(uoW, repository, new BllDalMapper<App.Domain.Service, App.BLL.DTO.Service>(mapper))
    {
    }
}

File: next-app1/src/services/ServiceService.ts
import { IService } from "@/domain/IService";
import { BaseEntityService } from "./BaseEntityService";
import { IJWTResponse } from "@/dto/IJWTResponse";

export class ServiceService extends BaseEntityService<IService> {
    constructor(setJwtResponse: ((data: IJWTResponse | null) => void)) {
        super('v1/Services', setJwtResponse);
    }
}